packages/p2p/src/components/orders/chat/chat-messages.jsx

```diff
@@ -27,7 +27,6 @@ const ChatMessages = observer(() => {
         if (sendbird_store.chat_messages.length > 0 && scroll_ref.current) {
             // Scroll all the way to the bottom of the container.
             scroll_ref.current.scrollTop = scroll_ref.current.scrollHeight;
-            sendbird_store.markMessagesAsRead();
```

Comment:
Removing this duplicate call since we are already handling this on scroll of chat container.